test/fixtures/mailer/gatekeeper_approval_feedback

```diff
@@ -1,7 +1,7 @@
 Hi Aaron Spiggle,
 
     Upon your publishing requests, the Sysmo SEEK -asset_gatekeeper- Gatekeeper Last, http://localhost:3000/people/-person_id- approved to publish:
-        Data file: Picture - http://localhost:3000/data_files/-df_id-
+        Datafile: Picture - http://localhost:3000/data_files/-df_id-
```

Comment:
Rather than change the fixture, can we change the mailer templates to use I18n?

(I guess they were relying on the inflections before)

Reply:
Should be done. I use model_name.human as it should concern models only. But I am not super familiar with the gatekeeper so don't hesitate to correct me :)